### PR TITLE
Bypass

### PR DESCRIPTION
**Pull request by siwan-cb**

### CODE REVIEW

File: src/middleware.ts [hunk 1]
```diff
@@ -14,6 +16,13 @@ export function middleware(request: NextRequest) {
   const hasCookie = request.cookies.has(process.env.PASSWORD_COOKIE_NAME!);
   const url = request.nextUrl.clone();
 
+  const shoudlBypass = bypassUrls.some((url) =>
+    request.nextUrl.pathname.startsWith(url)
+  );
+  if (shoudlBypass) {
+    return;
+  }
```
Comment (siwan-cb) [inline src/middleware.ts:24]: This should let 3 endpoints bypass @amit-shah-cb

File: src/middleware.ts [hunk 2]
```diff
@@ -2,6 +2,8 @@ import { isAfter, isBefore } from 'date-fns';
 import { NextResponse } from 'next/server';
 import type { NextRequest } from 'next/server';
 
+const bypassUrls = ['/api/inngest', '/api/cron', '/api/webhook'];
```
Comment (amit-shah-cb) [inline src/middleware.ts:5]: also need to bypass `/frames` and `/api/frames`

File: src/middleware.ts [hunk 3]
```diff
@@ -37,7 +37,7 @@ export const config = {
     '/guild/:path*',
     '/leaderboard/:path*',
     '/levels/:path*',
-    '/api/:path*',
```
Comment (siwan-cb) [inline src/middleware.ts:40]: @amit-shah-cb

Comment (siwan-cb) [reply src/middleware.ts:40]: Screenshot 2024-02-14 at 10 09 20 PM

**Pull request merged**